### PR TITLE
build_packages: fix dependency loops

### PR DESCRIPTION
**Pull request by ajeddeloh**
Add more to the flags files to prevent systemd loops. Also support comma
syntax for arguments beyond the second argument.

### CODE REVIEW

**Comment by ajeddeloh** [comment]
Confirmed there are no more loops to fix.

File: build_packages [hunk 1]
```diff
@@ -227,7 +228,10 @@ break_dep_loop() {
 
 if [[ "${FLAGS_usepkgonly}" -eq "${FLAGS_FALSE}" ]]; then
   # util-linux[udev] -> virtual->udev -> systemd -> util-linux
-  break_dep_loop sys-apps/util-linux udev,systemd sys-apps/systemd cryptsetup
+  break_dep_loop sys-apps/util-linux udev,systemd \
+    sys-apps/systemd udev,cryptsetup \
```
Comment (dm0-) [inline build_packages]: There is no `udev` USE flag for systemd.

**Comment by ajeddeloh** [comment]
@dm0 it was breaking new sdk's created via `cork create` and `cork enter`, run locally. I'll play around and see if I can get a more minimal set of flags here, hopefully only on `util-linux`.

**Comment by dm0-** [comment]
What are the exact commands you run to reproduce it?  I've been wiping out Jenkins workspaces lately, so just about every nightly build starts with a fresh `cork` SDK for every job.

**Comment by ajeddeloh** [comment]
```
mkdir coreos-test
cd coreos-test
cork create
cork enter
./setup_board --board=amd64-usr --default
./build_packages
```
I'll try again and see if the latest alpha being released fixed it.

**Comment by dm0-** [comment]
I ran those commands just now and did not hit any errors.

**Comment by ajeddeloh** [comment]
I also tried just now with (another) fresh sdk and it worked fine. But I definitely hit this and @euank repro'd. I guess I'll close this and reopen if it happens again?

**Pull request closed without merging**